### PR TITLE
Support oneOf keyword

### LINKED ISSUE #5: `oneOf` is not supported

### PR DESCRIPTION
**Pull request by rbruggem**
Fixes #5.

Examples:
Screenshot 2022-02-04 at 15 20 06

Screenshot 2022-02-04 at 15 20 18

### CODE REVIEW

File: src/handlebarHelpers.ts [hunk 1]
```diff
@@ -56,6 +56,9 @@ Handlebars.registerHelper('refToResourceLink', (ref) => {
 })
 
 function refToResourceName(ref: string): string | undefined {
+    if (ref === undefined) {
```
Comment (jstasiak) [inline src/handlebarHelpers.ts:59]: `ref` can't be undefined if we're to trust the function signature, should we change the signature here?

File: src/markdownAdapters.ts [hunk 2]
```diff
@@ -66,6 +66,26 @@ function resolveAllOf(
...
+    schemaObject: OpenAPIV3.SchemaObject,
+    refs: IRefs,
+    depth: number | undefined = undefined,
+    depthCounter: number = 0,
+): OpenAPIV3.SchemaObject {
+    if (!schemaObject.oneOf) {
+        return schemaObject
+    }
+
+    schemaObject.oneOf = schemaObject.oneOf.map(
```
Comment (jstasiak) [inline src/markdownAdapters.ts]: Do we *need* to modify `schemaObject` in place? Maybe we could `return {...schemObject, oneOf: schemaObject.oneOf.map(...)}`?

File: src/handlebarHelpers.ts (hunk 1 above)
Comment (rbruggem) [reply src/handlebarHelpers.ts:59]: yup

**Comment by rbruggem** [comment]
@jstasiak both comments have been addressed.

**Review by jstasiak — APPROVED** [review]
Very nice!

**Pull request merged**